app/assets/stylesheets/common/base/edit-category.scss

```diff
@@ -176,6 +176,14 @@ div.edit-category {
       margin-bottom: 1em;
     }
   }
+
+  .edit-category-tab-topic-template {
+    .control-group .desc {
+      margin-top: 0.25rem;
+      font-size: var(--font-down-1);
+      color: var(--primary-medium);
```

Comment:
Nitpick, but we probably shouldn't add specific styling for such a particular case. Maybe there's a general purpose classname we can reuse?

Reply:
Can't seem to find an existing class. The closest is `.desc` from the settings UI but we would need to change the markup as the selector for that is scoped by `.settings .setting .desc`.

Reply:
Ok, no worries. We can leave the styles in for now.